### PR TITLE
feat: add RootActivity intent test

### PR DESCRIPTION
**Pull request by ducklin5**
#### Summary of changes/features added:
- adds RootActivity intent test
- fixes the last menu item from "My profile" to "My Profile" .. capital P
- adds an id to the My Requests layout (for testing purposes)

## Pull Request Checklist
- [ ] Unit test written?
- [ ] (**For later in project**) Integration tests written?
- [x] Java docs written?

### CODE REVIEW

File: client/app/src/androidTest/java/com/example/criengine/RootActivityTest.java [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+
+        // Asserts that the current activity is the RootActivity.
+        solo.assertCurrentActivity("Wrong Activity", RootActivity.class);
+
+        viewPager = solo.getCurrentActivity().findViewById(R.id.view_pager);
+        bottomNav = solo.getCurrentActivity().findViewById(R.id.bottom_navigation);
+    }
+
+    /**
+     * Tests that the activity starts on the right page
```
Comment (AdityaHarvi) [inline client/app/src/androidTest/java/com/example/criengine/RootActivityTest.java:59]: This is already tested for the login activity

**Review by AdityaHarvi — CHANGES_REQUESTED** [review]

File: client/app/src/androidTest/java/com/example/criengine/RootActivityTest.java (hunk 1 above)
Comment (ducklin5) [reply client/app/src/androidTest/java/com/example/criengine/RootActivityTest.java:59]: this tests that the correct viewpager page shows up *and* also tests the bottom navigation is at the right position. They are similar tests but not the same. No such thing as too much testing :P

Comment (rwmilford) [reply client/app/src/androidTest/java/com/example/criengine/RootActivityTest.java:59]: are we still using user2 for testing or are we using the intentTestingUser profile?

Comment (AdityaHarvi) [reply client/app/src/androidTest/java/com/example/criengine/RootActivityTest.java:59]: i suppooseeee

**Review by AdityaHarvi — APPROVED** [review]

**Review by rwmilford — COMMENTED** [review]
Just that one question, other than that, looks good

**Comment by AdityaHarvi** [comment]
Oh yeah, we swapped to using some other account. Take a look at sams latest PR

**Pull request merged**